### PR TITLE
Epistola Collegii fixes

### PR DESCRIPTION
**Pull request by kdmnk**
I made the article's various attributes to be alterable, set preview visible for users (also deleted the details on the tiny cards), added the news to the home page, and fixed bugs.

### CODE REVIEW

File: app/Http/Controllers/HomeController.php [hunk 1]
```diff
@@ -20,7 +23,15 @@ class HomeController extends Controller
      */
     public function index()
     {
-        return view('home', ['information' => DB::table('home_page_news')->first()->text]);
+
+        $response = Gate::inspect('view', EpistolaNews::class);
```
Comment (luksan47) [inline app/Http/Controllers/HomeController.php:27]: Why not use `Auth::user()->can('view', EpistolaNews::class);` like at other places?

File: app/Http/Controllers/StudentsCouncil/EpistolaController.php [hunk 2]
```diff
@@ -82,14 +82,13 @@ public function updateOrCreate(Request $request)
...
             'time' => 'nullable|date_format:H:i',
             'end_date' => 'nullable|date',
-            'further_details_url' => 'nullable|url',
-            'website_url' => 'nullable|url',
-            'facebook_event_url' => 'nullable|url|starts_with:https://www.facebook.com/events/',
-            'registration_url' => 'nullable|url|required_with:registration_deadline',
-            'registration_deadline' => 'nullable|date',
-            'fill_url' => 'nullable|url|required_with:registration_deadline',
-            'filling_deadline' => 'nullable|date',
-            'approved' => 'nullable|required_with:picture_upload|accepted', //BUG: will be required always (picture_upload always exists)
```
Comment (luksan47) [inline app/Http/Controllers/StudentsCouncil/EpistolaController.php:92]: This bug is fixed now?

File: resources/views/student-council/communication-committee/epistola.blade.php [hunk 3]
```diff
@@ -8,52 +8,24 @@
...
         @endif
     @endif
-    ">
+    " >
     <div class="card-image" style="overflow-x: auto;overflow-y: hidden;">
         <img class="materialboxed" id="{{$article->id}}" src="{{ $article->picture_path ?? url('/img/committee-logos/kommbiz.jpg')}}" onerror="standby({{$article->id}})">
     </div>
-    <div class="card-content activator">
-        <i class="material-icons float right" style="cursor: pointer">more_vert</i></span>
+    <div class="card-content" onclick="window.location='{{ route('epistola.preview')}}'"
```
Comment (luksan47) [inline resources/views/student-council/communication-committee/epistola.blade.php:15]: I'm not sure if that's the best way for redirecting, but ok.

**Review by luksan47 — APPROVED** [review]
LGTM w/ minor comments

File: resources/views/student-council/communication-committee/epistola.blade.php (hunk 3 above)
Comment (kdmnk) [reply resources/views/student-council/communication-committee/epistola.blade.php:15]: Using links (<a>tags) messed up the ui, that's why I used the Javascript method.

File: app/Http/Controllers/StudentsCouncil/EpistolaController.php (hunk 2 above)
Comment (kdmnk) [reply app/Http/Controllers/StudentsCouncil/EpistolaController.php:92]: Yes

**Pull request merged**